understanding/20/use-of-color.html

```diff
@@ -81,7 +81,29 @@ <h2>Intent of Use of Color</h2>
          </p>
 
       </div>
-      
+
+      <div class="note">
+
+         <p>Under certain conditions, most user agents provide users with a color-only cue that a link has been visited. However, due to several technical constraints, authors should not be held accountable for this use of color, and color-only indications of visited links should not result in a failure of Use of Color (or a number of other criteria). The technical constraints are as follows:
+            <ol>
+               <li> 
+                  User agents constrain the exposure of a link's visited state due to <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/Privacy_and_the_:visited_selector">privacy concerns</a>. Author queries to user agents will indicate all links have not been visited.
+               </li>
+               <li>
+                  Any available information on the visited state of a link would be inacurrate since it is browser-dependent. Even if an author could accurately get information on previously activitated links, the author would be constrained to the current browser's preserved history, and would be unable to determine if the user had visited the page using a different browser (or if the history was not preserved, either from cache clearing or use of private sessions).
```

Comment:
```suggestion
                  Any available information on the visited state of a link would be inaccurate since it is both user and browser-dependent. Even if an author could accurately get information on previously activated links by a certain user, the author would be constrained to the current browser's preserved history, and would be unable to determine if the user had visited the page using a different browser (or if the history was not preserved, either from cache clearing or use of private sessions).
```